create_report.R

```diff
@@ -0,0 +1,39 @@
+library(rmarkdown)
+
+working_dir = getwd()
+message(paste("Working directory:", working_dir))
+
+args = commandArgs(trailingOnly = TRUE)
+
+input_prefix = args[1]
+annot_filename = args[2] # ex: annotation.txt
+sampleid = args[3]
```

Comment:
nit : I'd personally call this sample_id with the conventions you are using

Reply:
Agreed, fixed